src/main/java/com/baran/charactersheetcreator/WebController.java

```diff
@@ -0,0 +1,32 @@
+package com.baran.charactersheetcreator;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Imports shouldn't be catch-alls. Try to import each dependency individually.  

Bonus: Why is this approach better than star imports?

Reply:
fixed that and turned off automatic star imports in InteliJ. (after 3 static imports from the same package InteliJ did a star import)

"Bonus: Why is this approach better than star imports?"
Because it creates better readability and prevents naming conflicts with other classes.